src/core/core.tooltip.js

```diff
@@ -810,9 +811,13 @@ var exports = Element.extend({
 			ctx.fillStyle = textColor;
 			helpers.each(bodyItem.before, fillLineOfText);
 
-			helpers.each(bodyItem.lines, function(line) {
+			len = bodyItem.lines.length;
+			for (j = 0; j < len; j++) {
```

Comment:
nit: it'd be more consistent with this codebase to do:
```
			lines = bodyItem.lines;
			for (j = 0, len = lines.length; j < len; j++) {
```

Reply:
done